shapiq/games/imputer/marginal_imputer.py

```diff
@@ -46,26 +53,25 @@ def __init__(
         sample_replacements: bool = True,
         sample_size: int = 100,
         categorical_features: list[int] = None,
+        joint_marginal_distribution: bool = False,
```

Comment:
Maybe adding an small docs for this would be nice.

Reply:
done